src/sql/src/func.rs

```diff
@@ -3493,6 +3511,39 @@ pub static MZ_INTERNAL_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(
         "mz_any" => Aggregate {
             params!(Any) => AggregateFunc::Any => Bool, oid::FUNC_MZ_ANY_OID;
         },
+        // Note: See "avg_internal_v1" for why this duplicate function exists.
+        //
+        // TODO(parkmycar): Once there are no customers with objects using `avg_internal_v1` we can
+        // delete this function.
+        "mz_avg_promotion_internal_v1" => Scalar {
```

Comment:
I don't see anything here that differs from mz_avg_promotion. Do we need both? I think plan_avg_internal_v1 can use mz_avg_promotion.